.env.example

```
@@ -13,7 +13,7 @@ API_LOGGING_ENABLED=true
 # --- Ollama settings ---
 OLLAMA_HOST=http://ollama:11434
 # OLLAMA_HOST=http://host.docker.internal:11434
-BUILT_IN_OLLAMA_MODEL=qwen3:0.6b
+BUILT_IN_OLLAMA_MODELS=qwen3:0.6b,gemma3:270m
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The BUILT_IN_OLLAMA_MODELS key should go before the OLLAMA_HOST key

(UnorderedKey)